test/ci_workflow_gen.go

```diff
@@ -121,16 +120,17 @@ var (
 		"xb_recovery",
 		"resharding",
 		"resharding_bytes",
+		"mysql57",
 		"mysql80",
 		"vreplication_across_db_versions",
 		"vreplication_multicell",
 		"vreplication_cellalias",
 		"vreplication_basic",
-		"vreplication_v2",
 		"vtorc",
 		"vtorc_8.0",
 		"schemadiff_vrepl",
 		"topo_connection_cache",
+		"vreplication_v2",
 	}
 
```

Comment:
The ordering of `vreplication_v2` was correct before

Reply:
Actually no, the vtorc one are also wrong

Reply:
so you want me to restore to original or not ??

Reply:
moved it to original place